README.md

```diff
@@ -24,10 +24,11 @@ Please submit any changes as pull requests to the development branch, all change
 ```
 nvm use 11.9.0
 npm install -g quasar-cli
-git clone https://github.com/loki-project/loki-electron-wallet
+git clone https://github.com/loki-project/loki-electron-gui-wallet
 cd loki-electron-wallet
-cp path_to_loki_binaries/lokid bin/
-cp path_to_loki_binaries/loki-wallet-rpc bin/
+// the binaries are what was downloaded in the prequisite step
```

Comment:
`//` comments usually don't work in command line, i'd rather we add an explanation above the code snippet saying `path_to_lokid_binaries` contains what was downloaded in the pre-requisite

Reply:
Maybe
- Download latest Lokid binaries and extract to a folder